rust/processor/src/processors/coin_processor.rs

```diff
@@ -30,13 +31,19 @@ use tracing::error;
 pub struct CoinProcessor {
     connection_pool: ArcDbPool,
     per_table_chunk_sizes: AHashMap<String, usize>,
+    deprecated_tables: TableFlags,
 }
 
 impl CoinProcessor {
-    pub fn new(connection_pool: ArcDbPool, per_table_chunk_sizes: AHashMap<String, usize>) -> Self {
+    pub fn new(
```

Comment:
Hmm do we need the flags in coin_processor and token_processor? I think we can tell them to stop running it and only run fungible_asset_processor and token_v2_processor, respectively.

Reply:
can `current_collection_datas` and `current_token_pending_claims` be replaced with any other tables? we mark them as No in [the doc](https://docs.google.com/spreadsheets/d/1gSuE27GsGKLCBLOxhlzonvHh_bzF5i1Uzx27Mf61uh0/edit?gid=0#gid=0)

Reply:
+ `transaction_metadata_processor`